ogr/services/abstract.py

```diff
@@ -56,15 +92,11 @@ def get_description(self):
         """
         raise NotImplementedError()
 
-    def pr_create(self, title, body, target_branch, current_branch):
+    def get_fork(self):
```

Comment:
Even though you say that properties should not access internet, I'd suggest to cache this since forks likely not change for a user.

Reply:
Do you mean caching the result forever? Or some "refresh=False" argument?

Do we want a property or method?

Reply:
good questions; I would probably only cache a real value, for None, I would keep retrying

let's leave it as it is for now and we can improve that in future